db/views/hmis_households_v01.sql

```diff
@@ -0,0 +1,30 @@
+-- To update, run rails generate scenic:view hmis_households --replace
```

Comment:
@eanders would appreciate if you can sanity-check this view before merging. It creates a table where each row is 1 household.

I just realized that the id needs to be changed since it's the primary key, to something truly unique like: `CONCAT(MAX("Enrollment"."data_source_id"), "Enrollment"."HouseholdID") AS id,`

Let me know if you spot anything else that looks off.

Reply:
Yep